src/auction.ts

```diff
@@ -120,6 +122,46 @@ declare module './events' {
     }
 }
 
+export interface AuctionOptionsConfig {
+    /**
+     * Specifies bidders that the Prebid auction will no longer wait for before determining the auction has completed.
+     * This may be helpful if you find there are a number of low performing and/or high timeout bidders in your page’s rotation.
+     */
+    secondaryBidders?: BidderCode[]
+    /**
+     * When true, prevents banner bids from being rendered more than once. It should only be enabled after auto-refreshing is implemented correctly. Default is false.
+     */
+    suppressStaleRender?: boolean;
+    /**
+     * When true, prevent bids from being rendered if TTL is reached. Default is false.
+     */
+    suppressExpiredRender?: boolean;
+}
+
+export interface PriceBucketConfig {
+    buckets: {
+        precision?: number;
+        max: number;
+        increment: number;
+    }[];
+}
+
+
+declare module './config' {
+    interface Config {
+        /**
+         * Since browsers have a limit of how many requests they will allow to a specific domain before they block,
+         * Prebid.js will queue auctions that would cause requests to a specific origin to exceed that limit.
+         * The limit is different for each browser. Prebid.js defaults to a max of 4 requests per origin.
+         */
+        maxRequestsPerOrigin?: number;
+        auctionOptions?: AuctionOptionsConfig;
+        priceGranularity?: (typeof GRANULARITY_OPTIONS)[keyof typeof GRANULARITY_OPTIONS];
+        customPriceBucket?: PriceBucketConfig;
+        mediaTypePriceGranularity?: {[K in MediaType]?: PriceBucketConfig} & {[K in VideoContext as `${typeof VIDEO}-${K}`]?: PriceBucketConfig};
```

Comment:
![I am the god of death](https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmxoZ3doa2w4YjNnOXZ5djJzcDdiNGIzeHJ1bzV5cmk1ZzB1YnBpaSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/QxNUlSv1uQvEFRiTmi/giphy.gif)

while writing that type definition